app/src/main/java/com/pajato/android/gamechat/main/MainActivity.java

```diff
@@ -164,8 +164,17 @@ public static void processSignIn(@NonNull final Intent intent) {
         View header = navView.getHeaderView(0) != null
             ? navView.getHeaderView(0)
             : navView.inflateHeaderView(R.layout.nav_header_main);
-        View layout = header.findViewById(R.id.currentProfile);
+
+        // Set onClickListeners for the different account icons.
+        View layout = header.findViewById(R.id.alternateAccountIcon1);
```

Comment:
Since you are doing this pattern four times, you could create a helper method for finding a view and adding "this" as a click listener.  Then call the helper method in an iterator, something like:

for (id in [R.id.alternateAccountIcon1, R.id.alternateAccountIcon2, R.id.alternateAccountIcon3, R.id.alternateAccountIcon4]) {
  addOnClickListener(id);
}